src/main/python/main.py

```diff
@@ -51,12 +93,14 @@ def __init__(self):
 
         # Create tab pages
         sweep_page = SweepPage()
-        cell_page = CellPage()
+        feature_page = FeaturePage()
+        plot_page = PlotPage()
 
         # Create tab widget
         tab_widget = QTabWidget()
         tab_widget.insertTab(0, sweep_page, "Sweeps")
-        tab_widget.insertTab(1, cell_page, "Cell")
+        tab_widget.insertTab(1, feature_page, "Features")
```

Comment:
I don't think we need to do this now since these are placeholders, but they should have more descriptive names. Maybe "cell features" and "response plots"?

Reply:
I am not sure about the more detailed names. The user would expect the features to be the "cell" features as user processes recording from a cell. Also, to me the "response" is a recording of cell response to a stimulus, while here we are also plotting features of the response: spike features, f-I curve.  Perhaps we should think more about it and modify then. But, I do not have a strong opinion.